modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,526 @@
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the trust store. |  |
+| Password a| String | The password used to protect the trust store. |  |
+| Type a| String | The type of store used. |  |
+| Algorithm a| String | The algorithm used by the trust store. |  |
+| Insecure a| Boolean | If true, no certificate validations will be performed, rendering connections vulnerable to attacks. Use at your own risk. |  |
+|===
+
+[[KeyStore]]
+=== Key Store
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the key store. |  |
+| Type a| String | The type of store used. |  |
+| Alias a| String | When the key store contains many private keys, this attribute indicates the alias of the key that should be used. If not defined, the first key in the file will be used by default. |  |
+| Key Password a| String | The password used to protect the private key. |  |
+| Password a| String | The password used to protect the key store. |  |
+| Algorithm a| String | The algorithm used by the key store. |  |
+|===
+
+[[standard-revocation-check]]
+=== Standard Revocation Check
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Only End Entities a| Boolean | Only verify the last element of the certificate chain. |  |
+| Prefer Crls a| Boolean | Try CRL instead of OCSP first. |  |
+| No Fallback a| Boolean | Do not use the secondary checking method (the one not selected before). |  |
+| Soft Fail a| Boolean | Avoid verification failure when the revocation server can not be reached or is busy. |  |
+|===
+
+[[custom-ocsp-responder]]
+=== Custom Ocsp Responder
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Url a| String | The URL of the OCSP responder. |  |
+| Cert Alias a| String | Alias of the signing certificate for the OCSP response (must be in the trust store), if present. |  |
+|===
+
+[[crl-file]]
+=== Crl File
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | The path to the CRL file. |  |
+|===
+
+[[Reconnection]]
+=== Reconnection
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> | The reconnection strategy to use. |  |
+|===
+
+[[reconnect]]
+=== Reconnect
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Frequency a| Number | How often in milliseconds to reconnect. | |
```

Comment:
(in milliseconds) format used earlier; suggest "How often to reconnect, in milliseconds."